tests/integ/scala/test_dataframe_join_suite.py

```diff
@@ -733,6 +765,10 @@ def test_using_joins(session, join_type, local_testing_mode):
     assert sorted(res, key=lambda x: -x[0]) == [Row(-1, -10), Row(-2, -20)]
 
 
+@pytest.mark.skipif(
+    "config.getvalue('local_testing_mode')",
+    reason="TODO: bug",
```

Comment:
did you do a quick triage of the failure (error information) or you will put some context into the doc?